src/tlog-test-timespec.c

```diff
@@ -120,27 +120,27 @@ main(void)
     TEST(all_sub, TS(1, 0), TS(0, 500000000), TS(0, 500000000));
     TEST(all_sub, TS(1, 500000000), TS(1, 0), TS(0, 500000000));
     TEST(all_sub, tlog_timespec_max, tlog_timespec_max, TS(0, 0));
-    TEST(all_sub, TS(1, 0), TS(0, 1), TS(0, 999999999));
+    TEST(all_sub, TS(1, 0), TS(0, 1), TS(0, TLOG_TIMESPEC_NSEC_PER_SEC));
```

Comment:
This and similar ones below are incorrect since 999999999 != 1000000000. Write these as `TLOG_TIMESPEC_NSEC_PER_SEC - 1` and `-TLOG_TIMESPEC_NSEC_PER_SEC + 1`.